api/types.proto

```diff
@@ -674,6 +674,15 @@ message RaftConfig {
 	uint32 election_tick = 5;
 }
 
+// EncryptionConfig defines the cluster-wide settings for encrypting data
+message EncryptionConfig {
+	// ManagerUnlockKey defines the key that locks a manager at rest.  This is the
+	// key encrypting key (KEK) that will encrypt the manager TLS keys at rest and
+	// the raft encryption keys at rest.  If not provided, then managers will be
+	// unlocked (will not require a key to start up from a shut down state).
+	bytes manager_unlock_key = 1;
```

Comment:
Oh good point, thanks.  I had forgotten that the KEK would be automatically generated.  Will fix.